SwiftUI View: Ellipse shape (#1292)
Codecademy/docs: content/swiftui/concepts/views/terms/ellipse/ellipse.md

---
Title: 'Ellipse'

> Description: 'Built-in ellipse shape that can be modified as needed.'

Reviewer: ```suggestion
Description: 'Creates an ellipse shape that can be modified as needed.'
```

Subjects:
  - 'Software Development'
  - 'Mobile Development'
  - 'Computer Science'
Tags:
  - 'SwiftUI'
  - 'SwiftUI Views'
  - 'Views'
CatalogContent:
  - 'learn-swiftui'
  - 'paths/build-ios-apps-with-swiftui'
---

> The **`Ellipse`** is one of SwiftUI pre-built shapes like rectangles, circle, capsule which can be modified as wanted.

Reviewer 2: ```suggestion
The **`Ellipse`** is one of SwiftUI pre-built shapes like rectangles, circles, and capsules which can be modified as wanted.
```

> The **`Ellipse`** is one of SwiftUI pre-built shapes like rectangles, circle, capsule which can be modified as wanted.

Author: How to I link the capsule shape if its not available online yet? only through my PR

> The **`Ellipse`** is one of SwiftUI pre-built shapes like rectangles, circle, capsule which can be modified as wanted.

Reviewer 2: @cilippofilia, Good point. Leave that unlinked for the moment since we don't know what order these are going to be merged.

## Syntax

```pseudo
var body: some View {
    Ellipse()

>     // modifiers

Reviewer: ```suggestion
        Modifiers here
```

}
```

## Example

The following example makes a blue `Ellipse` with the dimension of `150` wide x `50` height:

```swift
var body: some View {
    Ellipse()
        .fill(.blue)
        .frame(width: 150, height: 50)
}
```

This will display the following:

> ![Ellipse](https://raw.githubusercontent.com/Codecademy/docs/main/media/ellipse.png)

Reviewer: Thanks for renaming the file @cilippofilia! Is it possible to crop the image as well?
